chains/evm/cli/writer/writer_test.go

```diff
@@ -0,0 +1,68 @@
+package writer_test
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/chainbridge-core/chains/evm/cli"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/cli/writer"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/suite"
+)
+
+type WriterTestSuite struct {
+	suite.Suite
+}
+
+func TestRunTestSuite(t *testing.T) {
+	suite.Run(t, new(WriterTestSuite))
+}
+
+func (s *WriterTestSuite) SetupSuite()    {}
+func (s *WriterTestSuite) TearDownSuite() {}
+
+func (s *WriterTestSuite) TearDownTest() {}
+
+func (s *WriterTestSuite) TestWriteCliDataToFile() {
+	expectedLog := time.Now().Format("02-01|15:00.000 ") +
+		"Passed flags: gasLimit, gasPrice, help, jsonWallet, jsonWalletPassword, networkid, url,  with args: 7000000, 25000000000, false, test-wallet, test-wallet-password, 0, test-url, =>\n" +
+		"Called evm-cli with args: --gasLimit=\"7000000\" --gasPrice=\"25000000000\" --help=\"false\" --jsonWallet=\"test-wallet\" --jsonWalletPassword=\"test-wallet-password\" --networkid=\"0\" --url=\"test-url\" \n"
```

Comment:
Format: {09-01|14:30:21.165} {cmd and args used} => (new line) {finalized log}
example of finalaized log https://github.com/ChainSafe/chainbridge-core/blob/main/chains/evm/cli/erc20/deposit.go#L123

Idea is that WriteCliDataToFile should also accept some string that holds finalised logs, main idea of such a behaviour is to have stored resulted logs that could contain useful information, like addresses of deployed smart contracts or transaction hashes for debug